web/pages/semantics.md

```diff
@@ -0,0 +1,64 @@
+# Semantics in K
+
+<br>
+
+## Featured
+
+* **[C (Jul 2010 - Present)](https://github.com/kframework/c-semantics)**
+
+  K semantics of the [C programming language](<https://en.wikipedia.org/wiki/C_(programming_language)>). It can help catch the undefined behaviors of C programs.
+
+* **[KWasm (Aug 2015 - Present)](https://github.com/kframework/wasm-semantics)**
+
+  KWasm is the K semantics of [WebAssembly](https://webassembly.github.io/spec/).
+
+* **[Elrond (Aug 2015 - Present)](https://github.com/runtimeverification/elrond-semantics)**
+
+  The semantics of Elrond and Mandos.
+
+* **[IELE (Oct 2016 - Present)](https://ielelang.org/)**
+
+  Provides a model of IELE in K.
```

Comment:
```suggestion
* **[IELE (Oct 2016 - Present)](https://ielelang.org/)**

  IELE is the underlying VM integrated into the [Cardano](https://cardano.org/) blockchain. IELE is a register-based VM (inspired by LLVM), which attempts to avoid many of the missteps in design present in EVM.
```